classes/ObjectModel.php

```diff
@@ -923,13 +923,14 @@ public function validateFields($die = true, $error_return = false)
      * Checks if multilingual object field values are valid before database interaction.
      *
      * @param bool $die
-     * @param bool $error_return
+     * @param bool $errorReturn
      *
      * @return bool|string True, false or error message.
      * @throws PrestaShopException
      */
-    public function validateFieldsLang($die = true, $error_return = false)
+    public function validateFieldsLang($die = true, $errorReturn = false)
     {
+        $defaultLang = (int) Configuration::get('PS_LANG_DEFAULT');
```

Comment:
I didn't check the standard but I checked the whole file for other occurrences, I'll get used to it then, cheers